_extras/checkout.md

```diff
@@ -6,13 +6,30 @@ calendar: https://calendar.google.com/calendar/embed?src=oseuuoht0tvjbokgg3noh8c
 demopad: http://pad.software-carpentry.org/teaching-demos
 discussionpad: http://pad.software-carpentry.org/instructor-discussion
 ---
+
 After you have finished the two-day training class,
-you must go through these steps in order to complete your training and be certified:
+you must go through these steps in order to complete your training and be 
+fully certified as a Data and/or Software Carpentry instructor:
 
 1.  Submit a [small change to one of our lessons](#lesson-change).
 2.  Take part in an online [discussion session](#discussion-session).
 3.  Teach a short [demonstration lesson](#demo-lesson) online.
 
+These checkout steps aim to introduce you to key components of being a Software 
+and Data Carpentry instructor that are difficult to fully cover in the classroom 
+setting of the two-day training.  These are: 
+
+1.  Contributing to our lesson materials.  As our materials are developed by the 
+community, we want to ensure that all instructors know where and how to contribute 
+to our materials, particularly so that you can contribute in the future.  
```

Comment:
Yes and no.

I think we want them to know how to do this, because it enables them to contribute to the material, and this in turn ensures that they feel more ownership of the material. This is a benefit in itself IMO, but it can also help us ensure that people actually use our stuff (which they should, but you know, some people enjoy going their own way no matter what) because if needed they are able to modify it.

Second: if we send them off to do something with one of our lessons, that ensures that they actually read through them thoroughly and hopefully helps them figure out how they work from a teaching perspective. One thing I experienced running discussion sessions was a certain level of unconsciousness regarding how the lesson would play out in the real world, for instance in regard to which parts of the lessons that could be difficult to teach. If we get them to read (hopefully work through them) from a "how can I improve this" perspective, we might get them to interact more with the material.